api/core/v3/entityconfig.proto

```diff
@@ -0,0 +1,49 @@
+syntax = "proto3";
+
+import "github.com/gogo/protobuf@v1.3.1/gogoproto/gogo.proto";
+import "github.com/sensu/sensu-go/api/core/v2/meta.proto";
+import "github.com/sensu/sensu-go/api/core/v2/entity.proto";
+
+package sensu.core.v3;
+
+option go_package = "v3";
+option (gogoproto.populate_all) = true;
+option (gogoproto.equal_all) = true;
+option (gogoproto.marshaler_all) = true;
+option (gogoproto.unmarshaler_all) = true;
+option (gogoproto.sizer_all) = true;
+option (gogoproto.testgen_all) = true;
+
+// EntityConfig represents entity configuration.
+message EntityConfig {
```

Comment:
Quite a minor comment, but isn't it more stylistically correct (and consistent) to name this file `entity_config.proto`?